packages/admin/cms-admin/src/dam/helpers/getValidExtensionsForMimetype.ts

```diff
@@ -0,0 +1,14 @@
+import * as mimedb from "mime-db";
+
+export const getValidExtensionsForMimetype = (mimetype: string) => {
+    let supportedExtensions: readonly string[] | undefined;
+    if (mimetype === "application/x-zip-compressed") {
+        // zip files in Windows, not supported by mime-db
+        // see https://github.com/jshttp/mime-db/issues/245
+        supportedExtensions = ["zip"];
+    } else {
+        supportedExtensions = mimedb[mimetype]?.extensions;
```

Comment:
Using mimedb in the admin is, of course, not ideal since it increases the bundle size significantly

However, we already use it in [useDamFileUpload](https://github.com/vivid-planet/comet/blob/3ee4ce0963e4a63278addd1a51349708f2401863/packages/admin/cms-admin/src/dam/DataGrid/fileUpload/useDamFileUpload.tsx#L3), so it's not getting worse 😅. And it prevents API errors and improves UX

Reply:
We could consider lazy-loading it in the future. But we don't care about bundle size in the admin for now.